Markdown extensions reference (#113)
codewars/docs: content/references/markdown/extensions.md

```
````

If the active language of the user is TypeScript, the above Markdown is rendered to the following:

> ```typescript
> const add = (a: number, b: number) => a + b;
> ```

## Conditional Rendering

Conditional rendering blocks renders the content when the language selected by the user matches the condition specified in the info string.

### Basic syntax

````markdown
```if:java
For java, use `Preloaded.check(input)`.
```
````

If the selected language is java, this block is showing up in the description, rendered to:

> For java, use `Preloaded.check(input)`.

### Details

* The language name in the info string has to match a valid language ID. To find the language ID, visit its documentation from the [languages page](/languages/).
* To obtain proper rendering, language specific blocks need an empty line before and after them, just like code blocks.

> * You can use any markdown formatting you want inside conditional blocks.

Author: ```suggestion
* You can use any markdown formatting you want inside conditional blocks, but be careful [when inserting code fences](/references/markdown/extensions/#using-code-blocks-inside-conditional-blocks): prefere to use different kinds of code fences to distinguish them more easily.
```

> * You can use any markdown formatting you want inside conditional blocks, but be careful [when inserting code fences](/references/markdown/extensions/#using-code-blocks-inside-conditional-blocks): prefere to use different kinds of code fences to distinguish them more easily.

Author: ```suggestion
* You can use any markdown formatting you want inside conditional blocks, but be careful when inserting code blocks: prefere to use different kinds of code fences to distinguish them more easily (like `~~~` and ```` ``` ````: [see here](/references/markdown/extensions/#using-code-blocks-inside-conditional-blocks)).
```

> * You can use any markdown formatting you want inside conditional blocks, but be careful when inserting code blocks: prefere to use different kinds of code fences to distinguish them more easily (like `~~~` and ```` ``` ````: [see here](/references/markdown/extensions/#using-code-blocks-inside-conditional-blocks)).

Reviewer: ```suggestion
* You can use Markdown inside conditional blocks, but [be careful when inserting code blocks](/references/markdown/extensions/#using-code-blocks-inside-conditional-blocks).
```

```
````

If the active language is Ruby, the above renders:

> Hidden _if_ the active language is JavaScript **or** TypeScript.

### Using Code Blocks Inside Conditional Blocks

> To use code blocks within these conditional blocks, use tildes (`~`) to declare the conditional block or increase the number of backticks used.

Author: dunno, For a "reference", it seems interesting to me to give the information. But nothing imperative about that, yes. I remove it, so?

> To use code blocks within these conditional blocks, use tildes (`~`) to declare the conditional block or increase the number of backticks used.

Reviewer: Hmm, maybe we can just link to this section from

> * You can use any markdown formatting you want inside conditional blocks.

by adding some warning about using backtick code fences.

> To use code blocks within these conditional blocks, use tildes (`~`) to declare the conditional block or increase the number of backticks used.

Author: * added note in the bullet point above
* didn't change the section about "increasing the number of backticks"
